public/main.js

```diff
@@ -49,12 +49,33 @@ function reloadTodoList() {
     todoListPlaceholder.style.display = "block";
     getTodoList(function(todos) {
         todoListPlaceholder.style.display = "none";
+        var i = 0;
```

Comment:
not sure about using 'i' in this way, what is 'i' used for? is it required as an id for the delete button?